widget/check_test.go

```diff
@@ -96,3 +96,12 @@ func TestNewCheckWithData(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, false, v)
 }
+
+func TestCheck_SetText(t *testing.T) {
+	check := &widget.Check{Text: "Test"}
+	check.SetText("Other Text")
```

Comment:
I'm not sure what this line or the Refresh below do?

Reply:
I honestly don't know, this specific test I copied from the Label widget, I think there it has to do with render tests but I'm not sure. Deleting and committing again